src/riscv/lib/src/pvm/linux.rs

```diff
@@ -1009,6 +1013,101 @@ impl<M: ManagerBase> SupervisorState<M> {
         // Return 0 as an indicator of success
         Ok(0)
     }
+
+    fn write_rlimit<MC: MemoryConfig>(
```

Comment:
Usually having unassociated functions in an `impl` block is because you want to share type variables. This isn't really leveraged in this case though.

Reply:
That's also good.